src/background/services/background.ts

```diff
@@ -61,6 +62,30 @@ export class Background {
     this.sendToPopup.start();
   }
 
+  async injectPolyfill() {
+    try {
+      // TODO: When Firefox 128 is old enough, inject directly via manifest.
+      // Also see: injectPolyfill in contentScript
+      await this.browser.scripting.registerContentScripts([
+        {
+          world: 'MAIN',
+          id: 'polyfill',
+          allFrames: true,
+          js: ['polyfill/polyfill.js'],
+          matches: PERMISSION_HOSTS.origins,
+          runAt: 'document_start',
+        },
+      ]);
+    } catch (error) {
+      // Firefox <128 will throw saying world: MAIN isn't supported. So, we'll
+      // inject via contentScript later. Injection via contentScript is slow,
+      // but apart from WM detection on page-load, everything else works fine.
+      if (!error.message.includes(`world`)) {
+        this.logger.error(error);
```

Comment:
Lets add more context for this error. I'm thinking about something along these lines:

```
World `MAIN` not supported by your browser - {{ browser.details }}. Upgrading to Firefox version 128 is going fix this error. 

Error: {{ error }}
```

I assume that if someone is looking in the DevTools they will probably understand why this is happening.